composables/paginator.ts

```diff
@@ -13,7 +13,7 @@ export function usePaginator<T, P, U = T>(
   // so clone it
   const paginator = _paginator.clone()
 
-  const state = ref<PaginatorState>(isHydrated.value ? 'idle' : 'loading')
+  const state = shallowRef<PaginatorState>(isHydrated.value ? 'idle' : 'loading')
```

Comment:
Ah, I forgot about this one. I think we can revert this change, no?

Reply:
done